### DISCUSSION TITLE
[4.0.0] Clean Your Room

### DISCUSSION THREAD

**Discussion by trioderegion** [trioderegion/steinhardts-support-desk#37] [Steinhardt's Guide Updates]
### 4.0.1
* Increasing maximum dnd5e system version to < 4.0.0

## What's New
* Minimum `dnd5e` system version increased to `3.2.0`
* Verified for both FoundryVTT v11 and v12.
* Added transform context entries to owned Trick Weapons for quick access to transformations without "using" the trick weapon.
  ![image](https://github.com/trioderegion/steinhardts-support-desk/assets/14878515/71bcf774-49c6-43f9-a51c-4e38a3d2eba3)
* Added enchantment actions to class features which alter weapons (e.g. Rogue "Blade of Radiance" subclass's "Sanctified Champion" and "Final Judgement" feature)

### Trick Weapons
* Integrated into system-provided item subtype display within Actor inventories and removed from the "Container" row.
* Updated to attunement toggle on trick sheets.
* Nulling out trick weapon weight and type label for display.
* Data migration converting trickwep type.value to empty string for better display in inventory.
* Moving primary attunement status back to the trick weapon itself and removing attunement requirements from all form weapons.

### Class Features and Weapons
* Added default resource consumption targets (compendium UUID) to appropriate items (e.g. Channel Divinity and Firearm ammunition).
* Changed "Channel Divinity: Guardian Angel" to a summoning type action and created variations of "Guardian Angel" for radiant and necrotic damage types.
* Voracious Void radius scaling by level and scale value for damage die associated.
* Feature options, such as Warlock invocations, are now restricted properly by class level.
* Various data-entry corrections.

**Comment by EsurioJon** [comment]
Any word on having it ready for any version of dnd5e past 4.0? i updated not realizing it would affect me using the module :(

**Reply by trioderegion** [reply]
Work is progressing well, but it is still going to be a few more weeks before I can get everything upgraded. Thank you for your patience.
